README.md

```diff
@@ -62,6 +62,17 @@ git checkout 1.2
 
 Note that release branches might contain non-released patches.
 
+## GHC compatability
+|      | Linux | Windows | macOS |
+|------|-------|---------|-------|
+| 8.6  | ✔️     | ✔️      | ✔️     |
+| 8.8  | ✔️     | ❌      | ✔️     |
+| 8.10 | ✔️     | ✔️      | ❌     |
+| 9.0  | ✔️     | ✔️      | ✔️     |
+| 9.2† | ❌     | ❌      | ❌     |
+
+† GHC 9.2 contains a regression, rendering Clash error messages indecipherable. This change will be reverted in 9.4.
+
 ## Cabal
 To use Cabal you need both Cabal and GHC installed on your system. For Linux and MacOS users we recommend using [ghcup](https://www.haskell.org/ghcup/). Windows users are recommended to use the [Haskell Platform](https://www.haskell.org/platform/windows.html).
```

Comment:
This should be changed, since the Haskell Platform is deprecated

Reply:
I wonder if we should point users to https://www.haskell.org/downloads/ for instructions to install GHC if they don't have it

Reply:
I doubt GHCup will go anywhere, so I've left that in. But I do link to the downloads page too.